dotnetapp-dev/README.md

```diff
@@ -1,19 +1,23 @@
 # .NET Core Development Sample
 
-This .NET Core Docker sample demonstrates how to use Docker in your .NET Core development process. The sample works with both Linux and Windows containers.
+This .NET Core Docker sample demonstrates how to use Docker in your .NET Core development process. It includes multiple projects and unit testing. The sample works with both Linux and Windows containers.
 
-The [sample Dockerfile](Dockerfile) creates a .NET Core application image based off of the [.NET Core SDK Docker base image](https://hub.docker.com/r/microsoft/dotnet/). It builds and runs the application with the same image, which is a useful strategy for interative development but not optimal for production. Take a look at the [.NET Core Production Sample](../dotnetapp-prod/README.md) or [ASP.NET Core Production Sample](../aspnetapp/README.md) for production-oriented samples.
+The [sample Dockerfile](Dockerfile) creates an .NET Core application Docker image based off of the [.NET Core Runtime Docker base image](https://hub.docker.com/r/microsoft/dotnet/).
```

Comment:
1. Wording `an .NET Core` should be `a .NET Core`
2. I find the usage of `base` in `.NET Core Runtime Docker base image` to be confusing.  `base image` has a special meaning in Docker which the runtime image is not.  (comment applies to the next section as well)